src/google-optimize.js

```diff
@@ -15,12 +15,25 @@ registerFsReady(
     })
   });
 
+let lastExperimentID = undefined;
+let lastExperimentValue = undefined;
+let lastExperimentFired = undefined;
+
 // take the optimize callback and turn it into an FS event
-function optimizeCallback( value, name ) {
+function optimizeCallback( value, id ) {
   // "Experiment Viewed" was selected as of the time of this addition, it allows for a higher
-  // rate limit on calls.  Instead of 10/second 30/minute default, it allows 40/second 60/minute
+  // rate limit on calls.  Instead of 10/second 30/minute default, it allows 40/second 60/minut
+  let now = Date.now();
+  if( (lastExperimentID === id) && (lastExperimentValue === value) ){
+    if( lastExperimentFired && (now - lastExperimentFired < 1000) ){
```

Comment:
Could we make this configurable? This is probably an approach we need to generalize across examples.  Is there perhaps an easy way to pass in a POJO of well defined options to the integration that does so?